elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -3,36 +3,81 @@
-        <input class="form-check-input mt-0" type="radio"
-               name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
-               {{#checked}}checked{{/checked}} id="{{name}}-{{key}}">
-
-        <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
-            {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
-            <div class="ml-1 mr-1">{{{html}}}</div>
-        </label>
-
-        {{#display_feedback}}
-        <div class="pl-multiple-choice-feedback">
-            {{{feedback}}}
+
+{{#radio}}
+    {{#answers}}
+        <div class="form-check {{#inline}}form-check-inline d-inline-flex{{/inline}}{{^inline}}d-flex{{/inline}} align-items-center py-1">
+            <input class="form-check-input mt-0" type="radio"
+                name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
+                {{#checked}}checked{{/checked}} id="{{name}}-{{key}}">
+
+            <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
+                {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
+                <div class="ml-1 mr-1">{{{html}}}</div>
+            </label>
+
+            {{#display_feedback}}
+            <div class="pl-multiple-choice-feedback">
+                {{{feedback}}}
+            </div>
+            {{/display_feedback}}
+
+            {{#display_score_badge}}
+                {{#correct}}
+                    <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
+                {{/correct}}
+                {{#incorrect}}
+                    <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
+                {{/incorrect}}
+                {{#partial}}
+                    <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
+                {{/partial}}
+            {{/display_score_badge}}
         </div>
-        {{/display_feedback}}
-
-        {{#display_score_badge}}
-            {{#correct}}
-                <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
-            {{/correct}}
-            {{#incorrect}}
-                <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
-            {{/incorrect}}
-            {{#partial}}
-                <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
-            {{/partial}}
-        {{/display_score_badge}}
+    {{/answers}}
+{{/radio}}
+{{^radio}}
+<script>
+    $(function(){
+        $(document).ready(function(){
+            $(".pl-dropdown-choice-{{uuid}}.active").each(function(){
+                $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            });
```

Comment:
Any reason why this is done with Javascript instead of mustache?

Reply:
For some reason I thought this had to be done with Javascript, but I think you're right that it can be done with mustache (see [here](https://stackoverflow.com/questions/33726202/how-to-use-bootstrap-dropdown-as-an-html-form-select)).